dev/com.ibm.websphere.appserver.features/visibility/public/xmlWS-3.0/io.openliberty.xmlWS-3.0.feature

```diff
@@ -63,7 +63,14 @@ IBM-SPI-Package: \
  bin/xmlWS/tools/ws-wsgen.jar, \
  dev/spi/ibm/javadoc/io.openliberty.globalhandler.spi_1.0-javadoc.zip
 -jars=\
-  io.openliberty.globalhandler.spi; location:=dev/spi/ibm/
+ io.openliberty.com.sun.xml.messaging.saaj.2.0, \
+ io.openliberty.globalhandler.spi; location:=dev/spi/ibm/, \
+ io.openliberty.jakarta.xmlBinding.3.0; location:="dev/api/spec/", \
+ io.openliberty.jakarta.activation.2.0; location:="dev/api/spec/", \
+ io.openliberty.jakarta.jws.3.0; location="dev/api/spec/", \
+ io.openliberty.jakarta.soap.2.0; location="dev/api/spec/", \
+ io.openliberty.jakarta.xmlWS.3.0; location="dev/api/spec/", \
+ io.openliberty.xmlWS.3.0.internal.tools
```

Comment:
These are not needed here because they are already part of io.openliberty.xmlws.common-3.0 as bundles.  They should not be listed as both bundles and jars.